core/controllers/base_test.py

```diff
@@ -147,52 +142,21 @@ def _get_current_time(unused_cls):
             base.CsrfTokenManager, '_get_current_time',
             types.MethodType(_get_current_time, base.CsrfTokenManager)):
             # Create a token and check that it expires correctly.
-            token = base.CsrfTokenManager().create_csrf_token('uid', 'page')
+            token = base.CsrfTokenManager().create_csrf_token('uid')
             self.assertTrue(base.CsrfTokenManager.is_csrf_token_valid(
-                'uid', 'page', token))
+                'uid', token))
 
             current_time = orig_time + 1
             self.assertTrue(base.CsrfTokenManager.is_csrf_token_valid(
-                'uid', 'page', token))
+                'uid', token))
 
             current_time = orig_time + FORTY_EIGHT_HOURS_IN_SECS - PADDING
             self.assertTrue(base.CsrfTokenManager.is_csrf_token_valid(
-                'uid', 'page', token))
+                'uid', token))
 
             current_time = orig_time + FORTY_EIGHT_HOURS_IN_SECS + PADDING
             self.assertFalse(base.CsrfTokenManager.is_csrf_token_valid(
-                'uid', 'page', token))
-
-            # Check that the expiry of one token does not cause the other to
```

Comment:
This seems like a reasonable thing to still check if we're generating multiple tokens per page (e.g. one for the i18n footer and the other for the page itself)? Or should we consolidate so that there's at most one token per page?